crates/shared/src/price_estimation/native/coingecko.rs

```diff
@@ -124,25 +183,3 @@ mod observe {
         }
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
Since we cannot instantiate a valid web3 in unit test (maybe we could but it would require moving a lot of code around) and this test is not active in the CI (it is just for quick local check), I decided to remove it. We could consider moving web3 into unit tests too :thinking: